lib/jsonapi/serializer.ex

```diff
@@ -145,6 +145,14 @@ defmodule JSONAPI.Serializer do
     merge_related_links(data, info, remove_links?())
   end
 
+  defp merge_base_links(doc, data, view, conn) do
+    links =
+      %{self: view.url_for(data, conn)}
```

Comment:
Could we avoid using a pipeline here?

https://github.com/christopheradams/elixir_style_guide#bare-variables

Reply:
Done!

Reply:
Thanks 😁